components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -1578,22 +1577,9 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
 
         for (const repo of repositories) {
             const p = cloneUrlToProject.get(repo.cloneUrl);
-            const repoProvider = new URL(repo.cloneUrl).host.split(".")[0];
 
             if (p) {
-                if (p.userId) {
```

Comment:
@svenefftinge I'm not sure why is it alright to remove this case? Because the property is going to be removed?

Reply:
it seems it was added for https://github.com/gitpod-io/gitpod/pull/5128

Reply:
Sorry, I should have added a comment here. We have removed personal projects and this was a left over. I will go through the code and see if we can remove the userID from project all together in a separate PR.